frida_tools/lsd.py

```diff
@@ -19,9 +19,21 @@ def _start(self) -> None:
                 self._update_status(f"Failed to enumerate devices: {e}")
                 self._exit(1)
                 return
+            device_os = {}
+            device_name = {}
+            for device in devices:
+                device_name[device.id] = device.name
+                try:
+                    params = device.query_system_parameters()
+                    device_name[device.id] = params.get("name", device.name)
```

Comment:
The `name` is required so we shouldn't be defensive here.

Reply:
Hmm, I'm getting a `KeyError` without it. Looks like from one of my android devices.

```
{'arch': 'arm64', 'os': {'version': '11', 'id': 'android', 'name': 'Android'}, 'platform': 'linux', 'api-level': 30, 'access': 'jailed'}

  File "/Users/tmm1/fancybits/frida/frida-tools/frida_tools/lsd.py", line 29, in _start
    device_name[device.id] = params["name"]
KeyError: 'name'
```